src/app/storage/duck/utils.ts

```diff
@@ -3,12 +3,12 @@ const validateIP = /^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$/;
 const testS3Name = (value: string) => {
   if (value.length < 3 || value.length > 63) {
     return 'The bucket name can be between 3 and 63 characters long.';
-  } else if (!/^[-a-z0-9\.]+$/.test(value)) {
```

Comment:
Ah yeah good catch. There was a rule that was complaining about "unneeded escape characters that I have since turned off. I will revert this file back to its original state